src/test/java/ru/training/at/hw2/Ex2.java

```diff
@@ -0,0 +1,125 @@
+package ru.training.at.hw2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Ex2 {
+    String check;
+    private WebDriverWait webDriverWait;
+    WebDriver webDriver;
+    WebElement webElement;
+
+
+    @Test
+    public void ex2Point1() {  //1.Open Test Site by Url
+        System.setProperty("webdriver.chrome.driver",
+                "/Users/Nikita/Downloads/chromedriver_win32/chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts()
+                .implicitlyWait(10, TimeUnit.SECONDS);
+        webDriver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        Assert.assertEquals("https://jdi-testing.github.io/jdi-light/index.html",
+                webDriver.getCurrentUrl());
+    }
+
+    @Test
+    public void ex2Point2() {  //2.Assert Browser title
+        webElement = webDriver.findElement(By.xpath("//header"));
```

Comment:
Please follow Bys' priority
1. DOM (id, name)
2. CSS
3. XML
```suggestion
        webElement = webDriver.findElement(By.css("header"));
```